qdrant-landing/content/documentation/fastembed/fastembed-colbert.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Working with ColBERT 
+weight: 6
+---
+
+# How to Generate ColBERT Multivectors with FastEmbed
+
+With FastEmbed, you can use ColBERT to generate multivector embeddings. ColBERT is a powerful retrieval model that combines the strength of BERT embeddings with efficient late interaction techniques. FastEmbed will provide you with an optimized pipeline to utilize these embeddings in your search tasks.
+
+Please note that ColBERT requires more resources than other no-interaction models. We recommend you use ColBERT as a re-ranker instead of a first-stage retriever. 
+
+The first-stage retriever can retrieve 100-500 examples. This task would be done by a simpler model. Then, you can rank the leftover results with ColBERT.
+
+## Setup
+
+This command imports all late interaction models for text embedding.
+
+```python
+from fastembed import LateInteractionTextEmbedding
+```
+You can list which models are supported in your version of FastEmbed.
+
+```python
+LateInteractionTextEmbedding.list_supported_models()
+```
+This command displays the available models. The output shows details about the ColBERT model, including its dimensions, description, size, sources, and model file.
```

Comment:
maybe we could write somewhere that colbert generates an embedding for each token, otherwise it could be mistaken with the usual embedding (where we encode the whole sequence and the output is one embedding of dim X)